rb/lib/selenium/webdriver/common/socket_lock.rb

```diff
@@ -75,9 +70,8 @@ def can_lock?
       end
 
       def did_lock?
-        !!@server
+        @server
```

Comment:
because `#did_lock?` is a private method that is only used in `#lock` which is calling it such that a truthy value gives us the desired result and we should avoid !! when possible.